src/plugins/intel_cpu/src/emitters/snippets/x64/kernel_executors/brgemm.cpp

```diff
@@ -64,11 +64,22 @@ bool BrgemmKernelConfig::operator==(const BrgemmKernelConfig& rhs) const {
 }
 
 void BrgemmKernelConfig::update(dnnl_dim_t M, dnnl_dim_t N, dnnl_dim_t K, dnnl_dim_t LDA, dnnl_dim_t LDB, dnnl_dim_t LDC) {
-    m_M = M; m_N = N; m_K = K;
-    m_LDA = LDA; m_LDB = LDB; m_LDC = LDC;
+    // If M is zero, it means that Brgemm won't be executed (in Loop with work_amount = 0, for example)
```

Comment:
We already discussed it with Ivan and I fully agree with you!
However, seems like we have to use `empty` configs for these executors because you're right: `kernel executor table knows nothing about loops`.

Probably Ivan will tell more details about this situation as an author of this infrastructure 😃